packages/eslint-plugin/package.json

```diff
@@ -36,7 +36,8 @@
   "scripts": {
     "build": "unbuild",
     "typecheck": "tsc --noEmit",
-    "lint": "nr build && cd ./fixtures && eslint ./src"
+    "lint": "nr build && cd ./fixtures && eslint ./src",
+    "test": "mocha --loader=tsx \"tests/**/*.ts\" --reporter dot --timeout 60000"
```

Comment:
Why mocha instead of Vitest? 👀

Reply:
I have never seen vitest as a test framework for eslint packages yet
I will explore how to configure it

Reply:
Here is an example: https://github.com/eslint-stylistic/eslint-stylistic/blob/main/vitest.config.unit.ts#L8

Basically, you only need to turn globals on.

Reply:
ready

Reply:
thanks for advise )